static/app/components/searchQueryBuilder/index.tsx

```diff
@@ -328,11 +328,6 @@ const Wrapper = styled(Input.withComponent('div'))`
   position: relative;
   font-size: ${p => p.theme.fontSizeMedium};
   cursor: text;
-
-  :focus-within {
-    border: 1px solid ${p => p.theme.focusBorder};
-    box-shadow: 0 0 0 1px ${p => p.theme.focusBorder};
-  }
```

Comment:
Was this removal intentional? Just trying to understand if this is now covered elsewhere.

Reply:
yes, I added `:focus-within` to the styles of the `input` itself:

https://github.com/getsentry/sentry/blob/e5f9cfc1a554d61d5fd79bf228ea01ea751ad22e/static/app/components/core/input/index.tsx#L53-L59

the problem with an override like this in one-of components is that for `chonk`, we would need a different override, as we have a different implementation of the focus-ring.

I don’t think there’s any drawback with having an input apply it’s focus-ring to `:focus-within`, as it normally won’t have any elements inside of it.